docs/installation.rst

```diff
@@ -48,11 +45,9 @@ Execute the following command to install the latest stable version of Transforme
 
 .. code-block:: bash
 
-  # Execute one of the following commands
-  NVTE_FRAMEWORK=pytorch pip install git+https://github.com/NVIDIA/TransformerEngine.git@stable    # Build TE for PyTorch only. The default.
-  NVTE_FRAMEWORK=jax pip install git+https://github.com/NVIDIA/TransformerEngine.git@stable        # Build TE for JAX only.
-  NVTE_FRAMEWORK=tensorflow pip install git+https://github.com/NVIDIA/TransformerEngine.git@stable # Build TE for TensorFlow only.
-  NVTE_FRAMEWORK=all pip install git+https://github.com/NVIDIA/TransformerEngine.git@stable        # Build TE for all supported frameworks.
+  pip install git+https://github.com/NVIDIA/TransformerEngine.git@stable
+
+This will automatically detect any installed deep learning frameworks and build support for them. Set `NVTE_FRAMEWORK` in the environment (as a comma-separated list) to explicitly specify which frameworks to support.
```

Comment:
```suggestion
This will automatically detect supported deep learning frameworks installed and build TransformerEngine for them. Set `NVTE_FRAMEWORK` in the environment (as a comma-separated list) to explicitly specify which frameworks to support.
```

Reply:
What about:

This will automatically detect if any supported deep learning frameworks are installed and build Transformer Engine support for them. To explicitly specify frameworks, set the environment variable `NVTE_FRAMEWORK` to a comma-separated list.